test/bin/ci-travis-all.sh

```diff
@@ -1,3 +1,3 @@
 grunt $B0
-grunt test:karma --browsers bs_firefox_sierra,bs_chrome_sierra,bs_ie11_win81,bs_ie10_win81
+grunt test:karma --browsers $B1
 grunt test:karma --browsers bs_ie9_win7,bs_ie16_win10,bs_safari_10_3,bs_safari_11
```

Comment:
This _would_ be nice but as soon as `$B0` (nodeunit) tests end, Travis starts running the script on line 3. But given that line 2 has not finished yet, BrowserStack will hang, because we are allowed at most 5 tests in parallel. :(

Reply:
When running tests in a serial fashion, we hit travis (.org) limits of 50m per build. Sequence tested:
- nodeunit 
- bs_firefox_sierra,bs_chrome_sierra,bs_ie11_win81,bs_ie10_win81
- bs_ie9_win7,bs_ie16_win10,bs_safari_10_3,bs_safari_11